Info-Orbs/include/widgets/webDataElementCharacterModel.h

```diff
@@ -27,15 +27,15 @@ class WebDataElementCharacterModel: public WebDataElement {
     void setColor(String color);
     int32_t getColor();
 
-    void parseData(JsonObject doc);
-    void draw(TFT_eSPI& display, int32_t defaultColor, int32_t defaultBackground);
+    void parseData(const JsonObject& doc, int32_t defaultColor, int32_t defaultBackground) override;
+    void draw(TFT_eSPI& display) override;
 
    private:
     int32_t m_x = 0;
     int32_t m_y = 0;
-    String m_character;
+    String m_character = "";
```

Comment:
A `String` is an object that should self-initialize, so this is the exception that does not need to be initialized here.